assets/blocks/Bushes/Grape/GrapeBush.block

```diff
@@ -1,7 +1,7 @@
-{
-  "hardness": 2,
-  "mass": 100,
-  "translucent": true,
-  "doubleSided": true,
-  "tile": "SimpleFarming:GrapesBush"
-}
+{
+  "hardness": 2,
+  "mass": 100,
+  "translucent": true,
+  "doubleSided": true,
+  "tile": "SimpleFarming:GrapeBush"
+}

```

Comment:
That is such a simple error for me to now have to deal with pulling this code. :P